### PR TITLE
Add some help, remove some useless one

### PR DESCRIPTION
**Pull request by jvoisin**

### CODE REVIEW

**Comment by radare** [comment]
i like the concept of “useless help"

File: libr/core/cmd_print.c [hunk 1]
```diff
@@ -5006,22 +5026,7 @@ static int cmd_print(void *data, const char *input) {
...
-					" +-*/  math ops\n"
-					" |&^   bin ops\n"
-					" <<>>  shift ops\n"
-					" _J    jump\n"
-					" cJ    conditional jump\n"
-					" _C    call\n"
-					" _R    ret\n"
-					" ==    cmp/test\n"
-					" XX    invalid\n");
+			r_core_cmd_help (core, help_msg_pxA);
```
Comment (ret2libc) [inline libr/core/cmd_print.c]: indentation please :D

Comment (ret2libc) [reply libr/core/cmd_print.c]: and here I'd really like to say "run XXX and you'll fix it ;)"

**Comment by jvoisin** [comment]
I removed the help, because this is how the other r2 commands are behaving: we don't have much "terminal" commands accepting `?`:

```
[0x00000000]> pdf?
p: Cannot find function at 0x00000000
[0x00000000]> 
```

**Comment by ret2libc** [comment]
This should be handled with proper commands parsing...

**Pull request merged**